customization/plugins.md

```diff
@@ -70,6 +71,41 @@ to the local plug-in:
 More detailed information about overriding locale files is found in [Customizing text in ArchivesSpace](./locales.md)
 
 
+## Overriding the visual (web) presentation
+
+You can directly override any view file in the core application by placing an erb file of the same name in the analogous path.
+For example, if you want to override the appearance of the "Welcome" [home] page of the Public User Interface, you can make your changes to a file `show.html.erb` and place it at `plugins/my_fine_plugin/public/views/welcome/show.html.erb`. (Where *my_fine_plugin* is the name of your plugin)
+
+### Implementing a broadly-applied style or javascript change
+
+Unless you want to write inline style or javascript (which may be practiceable for a template or two), best practice is to create `plugins/my_fine_plugin/public/views/layout_head.html.erb` or `plugins/my_fine_plugin/frontend/views/layout_head.html.erb`, which contains the HTML statements to incorporate your javascript or css into the `<HEAD>` element of the template.  Here's an example:
+
+* For the public interface, I want to change the size of the text in all links when the user is hovering. 
+    - I create `plugins/my_fine_plugin/public/assets/my.css`:
+        ```css
+            a:hover {font-size: 2em;}
+         ```
+    - I create `plugins/my_fine_plugin/public/views/layout_head.html.erb`:
+      ```ruby
+      <link rel="stylesheet" media="all" href="/assets/my.css />
```

Comment:
Should this be something more like:

`<%= stylesheet_link_tag "#{@base_url}/assets/my.css", media: :all %>`

To match the example in [Adding CSS rules](https://github.com/archivesspace/tech-docs/blob/master/customization/theming.md)?

Otherwise, this looks good to me!